AllReadyApp/Web-App/AllReady/wwwroot/js/eventDetails.js

```diff
@@ -2,107 +2,107 @@
 (function (HTBox) {
     var EventDetailAdmin = (function () {
         function EventDetailAdmin() {
-            $("#messageVolunteersModal").on("show.bs.modal", function (e) {
+            $('#messageVolunteersModal').on('show.bs.modal', function (e) {
```

Comment:
These two files should not have been included. There are the solution to issue 1411 which I wanted to commit after this pull was accepted. Will commit them later.